### DISCUSSION TITLE
建议两个功能：拖动进度和评论区

### DISCUSSION THREAD

**Discussion by sash78** [the1812/Bilibili-Evolved#4716] [功能建议 / Ideas]
一是【鼠标悬停在视频卡片上拖动进度】和【画中画小窗拖动进度】，悬停拖动进度的功能旧版b站是有的，新版给砍掉了
二是 【评论按点赞数排序】，以前按热门排序就是直接按点赞数排序，现在也不行了，唉

**Comment by akira0245** [comment]
2跟前端关系不大了，我觉得做不到
